content/applications/finance/accounting/fiscal_localizations/localizations/chile.rst

```diff
@@ -384,7 +384,10 @@ You can manually change the document type if needed.
 .. image:: media/Chile15.png
    :align: center
    :alt: Document type selection on invoices.
-
+   
+.. important::
+   Documents type 33: Electronic Invoice must have at least one item with Tax, otherwise the SII
+   will Reject the document validation. 
```

Comment:
```suggestion
   Documents type 33: Electronic Invoice must have at least one item with tax, otherwise the SII
   will reject the document validation. 
```